doc/developer/design/20230126_designs_reviews.md

```diff
@@ -0,0 +1,99 @@
+# Scaling our design process
+
+## Summary
+
+<!--
+// Brief, high-level overview. A few sentences long.
+// Be sure to capture the customer impact - framing this as a release note may be useful.
+-->
```

Comment:
Do you plan to remove these comments once the docs leaves the draft stage? They are a bit in the way when reading the unrendered doc.

Reply:
yes!